operators/ehr_query_llm/zero_mq/publisher_op.py

```diff
@@ -0,0 +1,66 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from holoscan.core import Fragment, Operator, OperatorSpec
+
+from operators.ehr_query_llm.message_handling import MessageSender
+
+
+class ZeroMQPublisherOp(Operator):
+    """
+    Named inputs:
+        message: message to be published to 0MQ
+
+    Named output:
+        None
+    """
+
+    def __init__(
+        self,
+        fragment: Fragment,
+        *args,
+        topic: str,
+        queue_endpoint: str = "tcp://*:5556",
```

Comment:
Should this match app defaults? I see `5600` / `5601` documented in the `fhir` README